runtime/version/fork.go

```diff
@@ -25,11 +25,19 @@ var versionToString = map[int]string{
 	Deneb:     "deneb",
 	Electra:   "electra",
 	Fulu:      "fulu",
+	Gloas:     "gloas",
 }
 
 // stringToVersion and allVersions are populated in init()
 var stringToVersion = map[string]int{}
 var allVersions []int
+var supportedVersions []int
+
+// unsupportedVersions contains fork versions that exist in the enums but are not yet
+// enabled on any supported network. These versions are removed from All().
+var unsupportedVersions = []int{Gloas}
+
+var unsupportedVersionSet = map[int]struct{}{}
```

Comment:
You can get rid of the list and declare the keys directly in the map:
```
var unsupportedVersions = map[int]struct{}{
    Gloas: struct{}{}
}
```

Deletes this whole chunk of init code:
```
unsupportedVersionSet = make(map[int]struct{}, len(unsupportedVersions))
	for _, v := range unsupportedVersions {
		unsupportedVersionSet[v] = struct{}{}
	}
	sort.Ints(unsupportedVersions)
```

I think you can also drop the `Unsupported` func.